runbotics/runbotics-orchestrator-ui/src/translations/en/process.json

```diff
@@ -148,6 +148,20 @@
     "Process.Details.Modeler.Actions.BeeOffice.GetActivitiesByUrl.Output": "Output",
     "Process.Details.Modeler.Actions.BeeOffice.GetActivitiesByUrl.Variable": "Variable name",
     "Process.Details.Modeler.Actions.BeeOffice.GetActivitiesByUrl.VariableText": "Assign the output to variable",
+    "Process.Details.Modeler.Actions.Browser.Target.XPath": "XPath",
+    "Process.Details.Modeler.Actions.Browser.Target.CSS": "CSS",
+    "Process.Details.Modeler.Actions.Browser.Target.ID": "ID",
+    "Process.Details.Modeler.Actions.Browser.Target.ClassName": "Class",
+    "Process.Details.Modeler.Actions.Browser.Target.LinkText": "Link text",
+    "Process.Details.Modeler.Actions.Browser.Target.PartialLinkText": "Partial Link Text",
+    "Process.Details.Modeler.Actions.Browser.Target.None": "Custom",
+    "Process.Details.Modeler.Actions.Browser.Target.ID.Tooltip": "Selects an element by its id attribute (e.g. <strong>my-id</strong>",
+    "Process.Details.Modeler.Actions.Browser.Target.XPath.Tooltip": "Selects an element by its XPath (e.g. <strong>//div[@id='my-id']</strong>)",
+    "Process.Details.Modeler.Actions.Browser.Target.CSS.Tooltip": "Selects an element by its CSS selector (e.g. <strong>input[name='username']</strong>, <strong>css=#my-id</strong>)",
+    "Process.Details.Modeler.Actions.Browser.Target.ClassName.Tooltip": "Selects an element by its class name (e.g. <strong>my-class</strong>",
+    "Process.Details.Modeler.Actions.Browser.Target.LinkText.Tooltip": "Selects an link tag by its exact text (e.g. <strong>Click here</strong>)",
```

Comment:
maybe change it to `Selects a link html element by its exact text`
and rewrite the below one also